From f0dfa44f76c332b529d5c67ef8b2ba4639cddb09 Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@gentoo.org>
Date: Fri, 17 Aug 2012 17:34:44 -0500
Subject: [PATCH 1/2] libmatekbd: Require -lm when using sqrt.

Since we use sincos, sqrt, and atan, add an AC_SEARCH_LIBS([sqrt],[m]) so that it pulls in -lm where needed.

Signed-off-by: Steev Klimaszewski <steev@gentoo.org>
---
 configure.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.in b/configure.in
index b308af0..3320fc8 100644
--- a/configure.in
+++ b/configure.in
@@ -128,6 +128,8 @@ AC_ARG_ENABLE([tests],
 
 AM_CONDITIONAL(BUILD_TESTS, test "$enable_tests" = "yes")
 
+AC_SEARCH_LIBS([sqrt], [m])
+
 AC_OUTPUT([
 Makefile
 libmatekbd.pc
-- 
1.7.11.1

